src/game.cpp

```diff
@@ -3650,6 +3651,87 @@ bool game::save_player_data()
     }
 }
 
+void game::dump_stats( const std::string& what )
+{
+    load_core_data();
+    DynamicDataLoader::get_instance().finalize_loaded_data();
+
+    if( what == "GUN" ) {
+        std::cout
```

Comment:
You can pipe `stdout` to something else. I'm working on an automated tool for updating the wiki. I don't mind extending the dumper in a subsequent PR to write elsewhere but coming from the UNIX tradition I'd consider standard output to be the default solution